modules/fuse-box-responsive-api/index.js

```diff
@@ -301,28 +330,15 @@
 				/* @end */
 
 				if (data[2] && data[2].css === true) {
-					loadRemoteScript(path + data[2].name, true);
+					Promise.all([loadRemoteScript(path + data[2].name, true), new Promise((resolve, reject) => {
+						loadScript(path, data, $cache, id, resolve, reject)
+					})
+				]).then((values) => {
```

Comment:
Can you use `function` here? To keep the IE11 from breaking?

Reply:
Use whatever you need for it to work. not sure for ```function``` is. Does this code get babel'd because that would leave it up to the user to choose what they support via babel-env?

Reply:
Not I does't. It's your PR, you should fix it ;-)
```
then(function(values) => {
```

Simple is that.

Reply:
Didn't know you were talking about the arrow function. Sorry.